src/libcore/fmt/mod.rs

```diff
@@ -529,6 +529,29 @@ pub trait Debug {
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait Display {
     /// Formats the value using the given formatter.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::fmt;
+    ///
+    /// enum Foo {
+    ///     Bar,
+    ///     Stack,
+    /// }
+    ///
+    /// impl fmt::Display for Foo {
+    ///     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    ///         match *self {
+    ///             Foo::Bar => write!(f, "Foo::Bar"),
+    ///             Foo::Stack => write!(f, "Foo::Stack"),
```

Comment:
This is slightly confusing because it looks like a `Debug` representation. How do you feel about getting rid of the `Foo::` in the outputted string?